### DISCUSSION TITLE
Can someone please fix 'TN' and 'El Nueve' in Argentinian channels? They haven't worked in a while

### DISCUSSION THREAD

**Discussion by ferialimiroski** [iptv-org/community#1164] [Q&A]
Can someone please fix 'TN' and 'El Nueve' in Argentinian channels? They haven't worked in a while

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#how-to-report-a-broken-stream
